soc/v2/gr716a/Kconfig

```diff
@@ -2,7 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_GR716A
-	bool "GR716A LEON3 fault-tolerant microcontroller"
+	bool
```

Comment:
we still need a Kconfig.soc which defines
```
config SOC_GR716A
       bool
```

Reply:
Fixed